Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Get_entity_alerts_by_MITRE_tactics_README.md

```diff
@@ -0,0 +1,51 @@
+This playbook is part of the ‘Cortex XDR by Palo Alto Networks’ pack. This playbook fetch alerts related to specific entity  from Cortex XDR , on a given timeframe, based on MITRE Tactics.
```

Comment:
lets change the "fetch" to other word, can be confusing for customer with Fetching Incidents (integration param)